fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -421,6 +422,13 @@ protected RelNode convertDruidQueryToRelNode(
             queryRelNode = convertDruidQueryWithFilteredAgg(builder, druidQuery, apiToFieldMapper, sqlTable);
         } else {
             builder = builder.scan(sqlTable.getName());
+            RelBuilder finalBuilder = builder;
+            List<RexNode> sketchMetrics = druidQuery.getAggregations().stream()
+                    .filter(aggregation -> aggregation.getType().equals("thetaSketch"))
```

Comment:
ThetaSketchAggregation.AGGREGATION_TYPE as the magic string here to support compatibility if things ever change.